developers/weaviate/configuration/inverted-indexes.md

```diff
@@ -0,0 +1,247 @@
+---
+title: Inverted indexes
+sidebar_position: 15
+image: og/docs/indexes.jpg
+# tags: ['configuration']
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import FilteredTextBlock from '@site/src/components/Documentation/FilteredTextBlock';
+import PyCodeV4 from '!!raw-loader!/_includes/code/howto/indexes/indexes-inverted-v4.py';
+import PyCodeV3 from '!!raw-loader!/_includes/code/howto/indexes/indexes-inverted-v3.py';
+import TSCodeV3 from '!!raw-loader!/_includes/code/howto/indexes/indexes-inverted-v3.ts';
+import TSCodeV2 from '!!raw-loader!/_includes/code/howto/indexes/indexes-inverted-v2.ts';
+
+Weaviate uses [inverted indexes](/developers/weaviate/concepts/indexing#inverted-indexes), also known as keyword indexes, to make textual and numeric searches more efficient. Weaviate provides different kinds to inverted index so you can match better match the index to your data.
+
+These indexes are normally configured on a property level:
+
+- [indexSearchable](#indexSearchable)
+- [indexFilterable](#indexfilterable)
+- [indexRangeFilters](#indexrangefilters)
+
+To tune inverted indexes at the collection level, adjust these configuration settings:
+
+- [BM25 search algorithm](#bm25)
+- [indexNullState](#collection-level-properties)
+- [indexPropertyLength](#collection-level-properties)
+- [indexTimestamps](#collection-level-properties)
+
+## indexSearchable
+
+The `indexSearchable` index improves property search times. This index is enabled by default. [Keyword search](/developers/weaviate/search/bm25) and [hybrid search](/developers/weaviate/search/hybrid) use this index.
```

Comment:
> The `indexSearchable` index improves property search times.

Just tried disabling this; and I think BM25 searches do not work at all without it.